src/main/java/com/knighttodo/knighttodo/service/impl/TodoBlockServiceImpl.java

```diff
@@ -42,21 +54,25 @@ public TodoBlock findById(long todoBlockId) {
     }
 
     @Override
-    public TodoBlock updateTodoBlock(TodoBlock changedTodoBlock) {
+    public TodoBlockVO updateTodoBlock(TodoBlockVO changedTodoBlockVO) {
+
+        final TodoBlock todoBlock = todoBlockGateway.findById(todoBlockMapper.toTodoBlock(changedTodoBlockVO).getId())
```

Comment:
There is no use in "final" keyword here I suppose. It's an inner method of the service we shouldn't change the link to this object anyway.

Reply:
sure, will remove it